frontend/src/locale/index.ts

```diff
@@ -1913,6 +1968,58 @@ Affrettati a comprare il biglietto!`,
     "profile.myProfile": "Il mio profilo",
     "profile.myProposals": "Le mie proposte",
 
+    "profile.myGrant": "Il mio grant",
+    "profile.myGrant.nextSteps": "I prossimi passi",
+    "profile.myGrant.grantType": "Tipo di Grant",
+    "profile.myGrant.appliedFor": "Richiesto",
+    "profile.myGrant.appliedFor.ticket": "Biglietto",
+    "profile.myGrant.appliedFor.travel": "Viaggio",
+    "profile.myGrant.appliedFor.accommodation": "Alloggio",
+    "profile.myGrant.edit": "Modifica",
+    "profile.myGrant.editInfo":
+      "Assicurati che tutte le informazioni fornite siano corrette. Hai tempo fino al {editDeadline} per modificare il tuo grant.",
+    "profile.myGrant.manage": "Gestisci",
+    "profile.myGrant.manage.warning": `Se i tuoi piani cambiano o non puoi più partecipare alla conferenza, per favore faccelo sapere cliccando sul pulsante 'Gestisci' per rifiutare il tuo grant o inviaci un'e-mail a {grantsEmail} il prima possibile.
```

Comment:
same as above, "Se i tuoi piani cambiano o non puoi più partecipare alla conferenza" doesn't make sense to me

maybe let's rewrite the english one and then translate it with chatgpt?

Reply:
ho cambiato